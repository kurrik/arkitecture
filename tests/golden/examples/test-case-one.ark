# Test case one - basic node
node1 {
  label: "Test Node"
}