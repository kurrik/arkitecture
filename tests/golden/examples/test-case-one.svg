<svg xmlns="http://www.w3.org/2000/svg" width="78" height="28">
  <defs>
    <marker id="arrowhead" markerWidth="10" markerHeight="7" 
            refX="9" refY="3.5" orient="auto" markerUnits="strokeWidth">
      <polygon points="0 0, 10 3.5, 0 7" fill="black" />
    </marker>
  </defs>

  <!-- Node rectangles and labels -->
  <rect x="0" y="0" width="78" height="28" fill="white" stroke="black" stroke-width="1" />
  <text x="39" y="14" text-anchor="middle" dominant-baseline="middle" font-family="Arial" font-size="14">Test Node</text>
</svg>