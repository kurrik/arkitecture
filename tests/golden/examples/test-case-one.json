{
  "fontSize": 14,
  "fontFamily": "Arial"
}